xml/System.Net.WebSockets/WebSocketState.xml

```diff
@@ -68,7 +68,7 @@
       </ReturnValue>
       <MemberValue>6</MemberValue>
       <Docs>
-        <summary>Reserved for future use.</summary>
+        <summary>Indicates the WebSocket is closed prematurely without complete handshake.</summary>
```

Comment:
```suggestion
        <summary>Indicates that the WebSocket has been aborted.</summary>
```
AFAIU the code, this state is a result of calling [Abort](https://docs.microsoft.com/en-us/dotnet/api/system.net.websockets.websocket.abort?view=net-5.0) on the web socket.

Reply:
I didn't have time to do a clean testing but it seems the disconnection from counterparty also lets the WebSocket enter `Abort` state. Your suggestion holds nonetheless.